src/dotnet-interactive-vscode/src/vscode/notebookProvider.ts

```diff
@@ -10,7 +10,7 @@ import { CellOutput, ReportChannel } from '../interfaces/vscode';
 import { Diagnostic, DiagnosticSeverity, NotebookCell, NotebookCellDisplayOutput, NotebookCellErrorOutput, NotebookCellOutput, NotebookCellTextOutput, NotebookDocument } from './../contracts';
 import { getEol, isUnsavedNotebook, toVsCodeDiagnostic } from './vscodeUtilities';
 import { getDiagnosticCollection } from './diagnostics';
-import { isDisplayOutput, isErrorOutput, isNotNull, isTextOutput } from '../utilities';
+import { isDisplayOutput, isErrorOutput, isNotNull, isTextOutput, stringify } from '../utilities';
```

Comment:
Don't need `stringify` anymore in this file.

Reply:
good spotting it!